src/Components/Components/src/HotReload/HotReloadManager.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+
+[assembly: AssemblyMetadata("ReceiveHotReloadDeltaNotification", "Microsoft.AspNetCore.Components.HotReload.HotReloadManager")]
+
+namespace Microsoft.AspNetCore.Components.HotReload
+{
+    // Not to be confused with the HR Manager.
+    internal static class HotReloadManager
+    {
+        // Hot reload stuff
+        internal static bool IsHotReloadEnabled = Environment.GetEnvironmentVariable("COMPLUS_ForceEnc") == "1";
```

Comment:
Interesting. Is this temporary or permanent?

Reply:
Let me see if we could convince the runtime to add this as a capability.